src/qt/forms/aboutdialog.ui

```diff
@@ -80,7 +80,7 @@
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="text">
-          <string notr="true">0.1.2</string>
```

Comment:
This could be left as the original 0.1.2 versioning as this is dynamically changed upon startup of the wallet by the wallet itself. (Opinion)